libs/src/main/python/dlpx/virtualization/libs/libs.py

```diff
@@ -468,7 +468,15 @@ def retrieve_credentials(credentials_supplier):
     response = internal_libs.retrieve_credentials(credentials_request)
     response_to_str(response)
     credentials_result = _handle_response(response)
-    if credentials_result.password != "":
+    # As protobuf definition of credentials_result object has all the
+    # attributes private_key, public_key and password irrespective of
+    # whether it is a keypair or a password credential type, we consider the
+    # object to be password credential type if private_key and public_key is
+    # not set.
```

Comment:
for multiline comments add a `#` line before and after. so:
```
    #
    # As protobuf definition of credentials_result object has all the
    # attributes private_key, public_key and password irrespective of
    # whether it is a keypair or a password credential type, we consider the
    # object to be password credential type if private_key and public_key is
    # not set.
    #
```

Reply:
I've made changes as suggested.